docs/getting-started/offline-endpoint.asciidoc

```diff
@@ -0,0 +1,180 @@
+
+[discrete]
+._Example: Nginx_
+[%collapsible]
+====
+Below is an example script that starts an Nginx Docker image and configures it as a file server:
+
+[source,sh]
+----
+cat > nginx.conf << EOF
+server {
+  root /app/static;
+  location / {}
+}
+EOF
+docker run --name artifactserver -v "$PWD"/nginx.conf:/etc/nginx/conf.d/default.conf:ro -v "$PWD"/static:/app/static:ro -p 80:80 -d nginx
+----
+
+IMPORTANT: This example script is not appropriate for production environments. We recommend configuring the Nginx server to use http://nginx.org/en/docs/http/configuring_https_servers.html[TLS] according to your IT policies. Refer to https://docs.nginx.com/nginx/admin-guide/installing-nginx/[Nginx documentation] for more information on downloading and configuring Nginx. 
+====
+
+[discrete]
+._Example: Apache HTTPD_
+[%collapsible]
+====
+Below is an example script that starts an Apache httpd Docker image and configures it as a file server:
+
+[source,sh]
+----
+docker run --name artifactserver -p 80:80 -v "$PWD":/usr/local/apache2/htdocs/ httpd
+----
+
+IMPORTANT: This example script is not appropriate for production environments. We recommend configuring httpd to use https://httpd.apache.org/docs/trunk/ssl/ssl_howto.html[TLS] according to your IT policies. Refer to https://httpd.apache.org[Apache documentation] for more information on downloading and configuring Apache httpd.
+====
+
+[discrete]
+=== Step 2: Configure {elastic-endpoint}
+
+Set the `advanced.artifacts.global.base_url` advanced setting for each <<configure-endpoint-integration-policy,{elastic-defend} integration policy>> that needs to use the mirror. Note that there's a separate setting for each operating system:
+
+* `linux.advanced.artifacts.global.base_url`
+* `mac.advanced.artifacts.global.base_url`
+* `windows.advanced.artifacts.global.base_url`
+
+[role="screenshot"]
+image::images/offline-adv-settings.png[Integration policy advanced settings]
+
+[discrete]
+=== Step 3: Manually copy artifact updates
+
+Download the most recent artifact files from the Elastic global artifact server, then copy those files to the server instance you created in step 1.
+
+Below is an example script that downloads all the global artifact updates. There are different artifact files for each version of {elastic-endpoint}. Change the value of the `ENDPOINT_VERSION` variable in the example script to match the deployed version of {elastic-endpoint}.
+
+[source,sh,subs="attributes"]
+----
+export ENDPOINT_VERSION={version} && wget -P downloads/endpoint/manifest https://artifacts.security.elastic.co/downloads/endpoint/manifest/artifacts-$ENDPOINT_VERSION.zip && zcat -q downloads/endpoint/manifest/artifacts-$ENDPOINT_VERSION.zip | jq -r --arg root "https://artifacts.security.elastic.co" '.artifacts | to_entries[] | $root + .value.relative_url' | xargs wget -P downloads/endpoint -c
```

Comment:
FYI: I swapped in a `{version}` variable here, so the version displayed in published docs will be based on the version that the reader is looking at. For the PR preview, that's `8.7.0` because the PR is based on `main`/`master`. 

We typically use variables like this for scripts/terminal examples in docs, but if this would be better hard-coded as `8.0.0`, I can change it back.

Reply:
I'm a +1 for the `{version}` label. I wonder why this source doesn't have a copy/paste option for the script like `[source, console]`?

Reply:
The `[source,console]` format adds copy options specifically for Kibana's Dev Tools Console, not a general copy/paste for command lines ([our Asciidoc docs](https://github.com/elastic/docs#copy-as-curlview-in-console) on this). I don't know if our stylesheet has general copy/paste options. The format on this page (`[source,sh]`) seems to display these commands most clearly (other formats like `[source,terminal]` gray out a lot of text).